torch/quantization/fx/quantization_patterns.py

```diff
@@ -927,7 +929,12 @@ class CopyNode(QuantizeHandler):
     def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 is_reference: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
-        return quantizer.quantized_graph.node_copy(node, load_arg(quantized=None))
+        qconfig = quantizer.qconfig_map[node.name]
+        if activation_is_int8_quantized(qconfig):
+            return quantizer.quantized_graph.node_copy(node, load_arg(quantized=None))
+        else:
+            # fp16 static quantization
+            return quantizer.quantized_graph.node_copy(node, load_arg(quantized=False))
```

Comment:
wondering on what's a good way to explain this?  Does `quantized` from `load_arg` mean the same thing as `activation_is_int8_quantized`?  If not, should we rename the `load_arg` argument to be more clear?

Reply:
yeah this is a bit confusing, I'll think about how to make this more clear